mesonbuild/interpreter/interpreterobjects.py

```diff
@@ -974,6 +974,10 @@ def process_method(self,
             FeatureNew.single_use(
                 f'Calling generator.process with CustomTaget or Index of CustomTarget.',
                 '0.57.0', self.interpreter.subproject)
+        elif any(isinstance(a, (build.Target)) for a in args[0]):
```

Comment:
CustomTarget is Target, so you'll end up with a false positive here

Reply:
The false positive is caught by the if statement above it, but I see now that there is a possible false negative if there is also a CustomTarget, CustomTargetIndex or GeneratedList in the  argument list. I will fix it in my next commit.